lib/katello.rb

```diff
@@ -12,6 +12,7 @@
 require "rest_client"
 require "anemone"
 require "securerandom"
+require "strong_parameters"
```

Comment:
We don't have the strong_parameters gem declared in the katello.gemspec anymore so this require statement will break our rails4 branch